src/gadm.jl

```diff
@@ -9,22 +9,145 @@ Please check its docstring for more details.
 module GADM
 
 using GeoIO
-using Meshes
 using GeoTables
+using TableTransforms
+using DataDeps
 
-import GADM as GADMData
+"""
+    GADM.isvalidcode(str)
 
+Tells whether or not `str` is a valid
+ISO 3166 Alpha 3 country code. Valid
+code examples are "IND", "USA", "BRA".
 """
-    GADM.get(country, subregions...; depth=0)
+isvalidcode(str) = match(r"\b[A-Z]{3}\b", str) !== nothing
+
+const API_VERSIONS = ("4.1", "4.0", "3.6", "2.8")
+
+"""
+    GADM.download(code; version="4.1")
+
+Downloads data for country `code` using DataDeps.jl and returns file path.
+The data is provided by the API of the [GADM](https://gadm.org) project.
+
+It is possible to choose the API version by passing it to the
+`version` keyword argument as string.
+The available API versions are: 4.1 (default), 4.0, 3.6 and 2.8.
+"""
+function download(country; version="4.1")
```

Comment:
```suggestion
function download(code; version=v"4.1")
```
Please use Julia's Version type.